api/apps/api/src/modules/access-control/projects-acl/project-acl.service.ts

```diff
@@ -41,6 +41,16 @@ export class ProjectAclService implements ProjectAccessControl {
     ProjectRoles.project_owner,
   ];
   private readonly canDeleteProjectRoles = [ProjectRoles.project_owner];
+  private readonly canExportProjectRoles = [
+    ProjectRoles.project_owner,
+    ProjectRoles.project_contributor,
+    ProjectRoles.project_viewer,
```

Comment:
double-check this guys - as far as I remember, this blocks editing scenarios so it may be something that blocks contributors/owners

Reply:
Would you mind to elaborate this? I don't really understand what do you mean 🤔 
These are just the roles on ACL allowed to request an export for a project

Reply:
Do you mean to be more restrictive with the allowed roles?

Reply:
just a generic thought @AaronPerezPerez - to reconsider who can export at all

Reply:
yep, we'll let viewers DDOS project/scenario editing, but I still think it's ok